kana_test.go

```diff
@@ -170,3 +170,19 @@ func TestNormalizeRomaji(t *testing.T) {
 		}
 	}
 }
+
+func TestHiraganaToKatakana(t *testing.T) {
+	for i, kana := range hiraganaList {
+		if got := HiraganaToKatakana(kana); got != katakanaList[i] {
+			t.Errorf("HiraganaToKatakana(%q) = %q, want %q", kana, got, katakanaList[i])
+		}
+	}
+}
```

Comment:
The test seems a little bit redundant to me, because it's using the same variables (`hiraganaList` and `katakanaList`) as the code it is testing. So it amounts to checking that the loop is correctly written, but there could be an underlying mistake in the slices, and we would not notice.

A better test would specifically write out the expected katakana and hiragana in the test file, and use that in the test. One long string with all the hiragana and another with all the katakana would suffice for this purpose. Then we could also check in other cases that the code correctly handles non-kana, empty strings, repeating kana, small kana, etc. Just my thoughts.